components/Layout/AudioControls.vue

```diff
@@ -1,135 +1,315 @@
+              <time-slider v-if="!isFullScreenPlayer" />
+            </div>
+          </v-col>
+          <v-col cols="3" class="d-none d-md-flex align-center justify-end">
             <v-btn class="d-none d-md-inline-flex" icon disabled>
               <v-icon size="18">{{
                 getCurrentItem.UserData.IsFavorite
                   ? 'mdi-heart'
                   : 'mdi-heart-outline'
               }}</v-icon>
             </v-btn>
-          </span>
-          <nuxt-link
-            v-if="getCurrentItem.AlbumArtists[0].Id"
-            tag="span"
-            class="text--secondary text-caption text-truncate mt-md-n2 link"
-            :to="`/artist/${getCurrentItem.AlbumArtists[0].Id}`"
+            <v-tooltip top>
+              <template #activator="{ on, attrs }">
+                <v-btn disabled icon class="mr-2" v-bind="attrs" v-on="on">
+                  <v-icon>mdi-playlist-play</v-icon>
+                </v-btn>
+              </template>
+              <span>{{ $t('queue') }}</span>
+            </v-tooltip>
+            <volume-slider />
+            <transition name="fade-fast" mode="in-out">
+              <v-tooltip v-if="!isFullScreenPlayer" top>
+                <template #activator="{ on, attrs }">
+                  <nuxt-link tag="span" :to="'/playback'">
+                    <v-btn icon class="ml-2" v-bind="attrs" v-on="on">
+                      <v-icon>mdi-fullscreen</v-icon>
+                    </v-btn>
+                  </nuxt-link>
+                </template>
+                <span>{{ $t('fullScreen') }}</span>
+              </v-tooltip>
+            </transition>
+            <v-tooltip top>
+              <template #activator="{ on, attrs }">
+                <v-btn
+                  icon
+                  class="ml-2"
+                  v-bind="attrs"
+                  v-on="on"
+                  @click="stopPlayback"
+                >
+                  <v-icon>mdi-close</v-icon>
+                </v-btn>
+              </template>
+              <span>{{ $t('stopPlayback') }}</span>
+            </v-tooltip>
+            <item-menu :item="getCurrentItem" :absolute="false" :dark="false" />
+          </v-col>
+          <v-col
+            cols="3"
+            class="d-flex d-md-none px-0 align-center justify-end"
           >
-            {{ getCurrentItem.AlbumArtist }}
-          </nuxt-link>
-        </div>
-      </v-col>
-      <v-col cols="6" class="pa-0 d-none d-md-inline">
-        <div class="d-flex flex-column justify-center">
-          <div class="d-flex align-center justify-center">
-            <v-btn disabled icon class="mx-1">
-              <v-icon>mdi-shuffle</v-icon>
-            </v-btn>
-            <v-btn icon class="mx-1" @click="setPreviousTrack">
-              <v-icon>mdi-skip-previous</v-icon>
-            </v-btn>
-            <v-btn icon large class="mx-1" @click="togglePause">
-              <v-icon large>
+            <v-btn
+              elevation="6"
```

Comment:
I think this looks better without elevation since none of the other buttons have it.

Reply:
@Camc314 I think it makes sense to make it more prominent since it's the main button of any music player 😋.

I agree with you in dark theme, but in light theme it also looks really plain without this.

Reply:
The light theme looks really weird with it elevation, imo. It's already emphasized by the fact that it's circled, too, so it feels a bit redundant.